src/notation/internal/notationinteraction.cpp

```diff
@@ -5442,6 +5447,7 @@ void NotationInteraction::addLyricsVerse()
         return;
     }
     mu::engraving::Lyrics* lyrics = toLyrics(m_editData.element);
```

Comment:
Looks like you can do a micro-refactor here, by immediately calling this `oldLyrics`, instead of calling it `lyrics` and then doing some swappery a few lines below